src/Tasks/SwitchSessionConfigTask.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Spatie\Multitenancy\Tasks;
+
+class SwitchSessionConfigTask implements SwitchTenantTask
+{
+    private $originalConnection;
+
+    private $originalDriver;
+
+    public function __construct()
+    {
+        $this->originalDriver = config('session.driver');
+        $this->originalConnection = config('session.connection');
+    }
+
+    public function makeCurrent($tenant): void
```

Comment:
Here I did not set the type of the parameter because I assumed that a custom tenant model could be used by the developer using the package.

Reply:
This causes problems with larastan, and other static analyzers because the original declaration does not match this line.